GRDB/Core/Database.swift

```diff
@@ -261,6 +263,36 @@ extension Database {
             throw DatabaseError(resultCode: code, message: String(cString: sqlite3_errmsg(sqliteConnection)))
         }
     }
+    
+    private static func set(cipherPageSize: Configuration.CipherPageSize, forConnection sqliteConnection: SQLiteConnection) throws {
+        var sqliteStatement: SQLiteStatement? = nil
+        let code = sqlite3_prepare_v2(sqliteConnection, "PRAGMA cipher_page_size = \(cipherPageSize.rawValue)", -1, &sqliteStatement, nil)
+        guard code == SQLITE_OK else {
+            throw DatabaseError(resultCode: code, message: String(cString: sqlite3_errmsg(sqliteConnection)))
+        }
+        defer {
+            sqlite3_finalize(sqliteStatement)
+        }
+        let step = sqlite3_step(sqliteStatement)
+        if step != SQLITE_DONE {
+            throw DatabaseError(resultCode: .SQLITE_MISUSE, message: "Unable to set cipher_page_size")
```

Comment:
`validateSQLCipher` throws SQLITE_MISUSE, because it checks for a programmer error. Here we can throw the same code as the one returned by `sqlite3_step`. After a renaming of `step` into `code`, this gives:

```suggestion
            throw DatabaseError(resultCode: code, message: "Unable to set cipher_page_size")
```

Reply:
Oops, we should reuse the standard message:

```suggestion
            throw DatabaseError(resultCode: code, message: String(cString: sqlite3_errmsg(sqliteConnection)))
```